### PR TITLE
add movies raw

### PR DESCRIPTION
**Pull request by GianCarloTG**

### CODE REVIEW

File: raw_movies.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    path_movie_id = f'./movies/raw_data/collections/'
+    raw_movies_id(path_movie_id)
+
+    movies_ids = get_movies_id(path_movie_id)
+    path_movie_info = './movies/raw_data/movies_info/'
+    path_movie_review = './movies/raw_data/movies_review/'
+    c = 1
+    for movie in movies_ids:
+        c +=1 # just a counter to check progress
+        if c == 1000:
```
Comment (DionisiusMayr) [inline raw_movies.py]: I guess this is only going to print once.
Shouldn't it be something like `c % 1000 == 0`?

**Review by DionisiusMayr — APPROVED** [review]
I think that you already collected some data, right? If so, LGTM

**Review by Matematikoi — APPROVED** [review]
Lgtm

**Review by DionisiusMayr — APPROVED** [review]
Bora

**Pull request merged**